lib/bolt/cli.rb

```diff
@@ -443,13 +471,21 @@ def execute(options)
       message = 'There may be processes left executing on some nodes.'
 
       if options[:mode] == 'plan'
-        executor = Bolt::Executor.new(@config, options[:noop], true)
+        unless options[:nodes].empty?
+          if options[:task_options]['nodes']
+            raise Bolt::CLIError,
+                  "plan's nodes parameter may be specified using the --nodes option, but in that case " \
```

Comment:
Based on formatting for other errors, I'd rework this to say: `A plan's 'nodes' parameter may be specified using the --nodes option, but in that case it must not be specified as a separate nodes=<value> parameter`.